edxval/api.py

```diff
@@ -230,6 +232,54 @@ def update_video_transcript(video_id, language, transcript_url, transcript_forma
     return TranscriptSerializer(transcript).data
 
 
+def get_3rd_party_transcription_plans():
+    """
+    Retrieves 3rd party transcription plans.
+    """
+    return THIRD_PARTY_TRANSCRIPTION_PLANS
+
+
+def get_transcript_preferences(course_id):
+    """
+    Retrieves course wide transcript preferences
+
+    Arguments:
+        course_id (CourseKey): a course key
+    """
+    transcript_preference = None
+    transcript_preference_set = TranscriptPreference.objects.filter(course_id=course_id)
```

Comment:
Why filter ? I think .get would be sufficient Or this is just to avoid exception?